robot_ws/src/hello_world_robot/package.xml

```diff
@@ -19,6 +19,7 @@
   <depend>visualization_msgs</depend>
   <depend>actionlib_msgs</depend>
   <depend>turtlebot3_msgs</depend>
+  <depend>turtlebot3_description</depend>
```

Comment:
This hasn't been changed to `turtlebot3_description_reduced_mesh`.

Reply:
@mm318 Added `turtlebot3_description_reduced_mesh` as dependency. Please take a look.